x-pack/plugins/security/server/routes/authorization/roles/get_all_by_space.ts

```diff
@@ -38,10 +40,52 @@ export function defineGetAllRolesBySpaceRoutes({
         const hideReservedRoles = buildFlavor === 'serverless';
         const esClient = (await context.core).elasticsearch.client;
 
-        const [features, elasticsearchRoles] = await Promise.all([
+        const [features, queryRolesResponse] = await Promise.all([
           getFeatures(),
-          await esClient.asCurrentUser.security.getRole(),
+          await esClient.asCurrentUser.security.queryRole({
+            query: {
```

Comment:
We want "All roles for that space including reserved and non-reserved roles". Is this query correct? 🤔